pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -150,6 +155,23 @@ func (d *DeviceCheck) Run(collectionTime time.Time) error {
 		d.sender.ReportMetrics(d.config.Metrics, values, tags)
 	}
 
+	// Get a system appropriate ping check
+	if d.config.PingEnabled {
+		log.Infof("ATTEMPTING TO RUN PING FOR HOST: %s, tags: %+v", d.config.IPAddress, tags)
+		pingResult, err := d.Ping(d.config.PingConfig)
+		if err != nil {
+			log.Errorf("%s: failed to ping device: %s", d.config.IPAddress, err.Error())
+			d.sender.Gauge(pingCanConnectMetric, float64(0.0), tags)
+		} else {
+			log.Infof("%s: ping returned: %+v", d.config.IPAddress, pingResult)
+			d.sender.Gauge(pingAvgRttMetric, float64(pingResult.AvgRtt/time.Millisecond), tags)
```

Comment:
The original unit we get back is nanoseconds (`time.Duration`).

Milliseconds made sense to me intuitively since most hosts I ping return in a matter of milliseconds, but I'm not an expert on the units being shown in the Datadog UI. Are you saying that we should send seconds as a `float64` because it would make displaying it easier?